invenio_records_marc21/ui/theme/assets/semantic-ui/js/invenio_records_marc21/uploads/components.js

```diff
@@ -0,0 +1,113 @@
+// This file is part of Invenio.
+//
+// Copyright (C) 2023 Graz University of Technology.
+//
+// Invenio-Records-Marc21 is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see LICENSE file for more
+// details.
+
+import React, { useState } from "react";
+import { truncate, get } from "lodash";
+import { Button, Card, Icon, Item, Label } from "semantic-ui-react";
+import { EditButton } from "../components/EditButton";
+
+export const Marc21RecordResultsListItem = ({ result, index }) => {
```

Comment:
I thought we have that already somewhere? Otherwise the search would not work?

Reply:
https://github.com/tu-graz-library/invenio-records-marc21/blob/master/invenio_records_marc21/ui/theme/assets/semantic-ui/js/invenio_records_marc21/search/components.js is this not the same?